resource_packs/packs.json

```diff
@@ -649,6 +649,11 @@
 					"name": "Mine Progress Bar",
 					"description": "Changes the breaking texture of blocks to a clean ring around the outside."
 				},
+				{
+					"id":"music_disc_redstone_level",
+					"name":"Music Disc Redstone Level",
+					"description":"Gives all Music Discs a redstone power level indicator in the tooltip."
```

Comment:
Pack should be after hunger preview

Reply:
why? I thought it was just alphabetical and VSC lists its alphabetically here